web/impact/impact/tests/test_user_list_view.py

```diff
@@ -108,16 +114,257 @@ def test_get(self):
             assert user2.email in emails
             assert user_count == response.data["count"]
 
-    def test_get_with_limit(self):
-        UserContext().user
-        UserContext().user
+    def test_get_returns_correct_count_attribute(self):
+        for _ in range(10):
+            UserContext()
```

Comment:
Why not `UserContext.create_batch(10)`?

Reply:
`UserContext` doesn't have a `create_batch` since it is not a subclass of `Factory`. and likewise.

Reply:
Ah. I was fooled by the fact that `JudgeFeedbackContext` has a `create_batch`; turns out it's manually defined on the class.